xdcr/rosmar_xdcr.go

```diff
@@ -373,3 +388,21 @@ func updateHLV(xattrs map[string][]byte, sourceHLV *db.HybridLogicalVector, sour
 	}
 	return nil
 }
+
+// getConflictResolutionCas returns cas for conflict resolution. If thethe conflict resolution cas.
```

Comment:
Typo here? I think it would be good to include on the comment the expected interaction between actual cas and mou cas, something like the below:
If we have:
- the _mou.cas == actualCas we will return _vv.cvCAS for conflict resolution, 
- if mou.Cas is out of date we will use _vv.ver
- if mou.Cas is not out of date but there is no HLV we return 0 to match XDCR